src/olympia/migrations/1121-update-db-schema-to-match-django.sql

```diff
@@ -0,0 +1,36 @@
+ALTER TABLE `abuse_reports`
+    MODIFY `created` datetime(6) NOT NULL,
+    MODIFY `modified` datetime(6) NOT NULL,
+    MODIFY `guid` varchar(255),
+    MODIFY `state` NOT NULL;
+
+ALTER TABLE `addons`
+    MODIFY `addontype_id` int(10) unsigned NOT NULL,
+    MODIFY `average_daily_users` int(10) unsigned NOT NULL,
+    MODIFY `averagerating` double,
+    MODIFY `bayesianrating` double NOT NULL,
+    MODIFY `created` datetime(6) NOT NULL,
+    MODIFY `current_version` int(10) unsigned,
+    MODIFY `description` int(10) unsigned,
+    MODIFY `developercomments` int(10) unsigned,
+    MODIFY `eula` int(10) unsigned,
+    MODIFY `homepage` int(10) unsigned,
+    MODIFY `icon_hash` varchar(8),
+    MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    MODIFY `inactive` tinyint(1) NOT NULL,
+    MODIFY `modified` datetime(6) NOT NULL,
+    MODIFY `name` int(10) unsigned,
+    MODIFY `last_updated` datetime(6) NOT NULL,
```

Comment:
This is the one that fails locally for me, and it would also fail on stage (probably dev & prod as well):
```sql
> SELECT * FROM addons WHERE last_updated IS NULL;
(...)
12250 rows in set (0.07 sec)
```

Reply:
oh, that's a mistake.  The django field definition is `last_updated = models.DateTimeField(       db_index=True, null=True,...)` so I shouldn't have added `NOT NULL` to it.